source/renderer/app/i18n/locales/en-US.json

```diff
@@ -101,6 +101,11 @@
   "daedalus.diagnostics.dialog.localTimeDifference": "Local time difference",
   "daedalus.diagnostics.dialog.localTimeDifferenceCheckTime": "Check time",
   "daedalus.diagnostics.dialog.localTimeDifferenceChecking": "Checking...",
+  "daedalus.diagnostics.dialog.meetsHardwareRequirementsStatus": "!!!Recommended system requirements status",
```

Comment:
the !!! is not required in `en-US.json` or `ja-JP.json`. the !!! is used to find untranslated strings from the manage:translations, so this will give false positives

Reply:
Translations are still in progress unfortunately